README.rst

```diff
@@ -25,20 +25,25 @@ That's it!
 Usage
 -----
 
-This site uses Jekyll_. The official docs are good, but here's a quickstart:
+Installation
+~~~~~~~~~~~~
+This site uses Jekyll_ and we've wrapped the development environment in Docker.
```

Comment:
Could we keep the jekyll instructions, and give people a choice to use jekyll or docker?